deployments/kustomize/cronjobs/BACKUP-update-current-year-cron.yaml

```diff
@@ -0,0 +1,216 @@
+# this is a backup of the working config for the update-current-year-cron CronJob.
```

Comment:
not sure why this file is even here?

Reply:
I plan to delete it soon. Sorry for the wasted time - it's just a backup in case the argo cd fails and I lose the original one.